### DISCUSSION TITLE
secret scanner doesn't match a secret_key

### DISCUSSION THREAD

**Discussion by ripperi8u** [aquasecurity/trivy#5645] [Bugs]
### Description

the content in a line like: aws_sec_key "KEYKEYKEYKEYKEYKEYKEYKEYKEYKEYKEYKEYKEYK"
it should be match with 'aws-secret-access-key',but it cannot match

### Desired Behavior

the content in a line like:   aws_sec_key "KEYKEYKEYKEYKEYKEYKEYKEYKEYKEYKEYKEYKEYK"
it should be match with 'aws-secret-access-key',but it cannot match

### Actual Behavior

It shoud be match with regex.I go through the file "pkg/fanal/secret/builtin-rules.go", found the Reusable regex patterns **connect**, change its value to ``\s*(:|=>|=)*\s*`` maybe work for this case 

### Reproduction Steps

```bash
1.
2.
3.
...
```

### Target

Filesystem

### Scanner

Secret

### Output Format

None

### Mode

None

### Debug Output

```bash
/
```

### Operating System

MacOS

### Version

```bash
0.47.0
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by DmitriyLewen** [comment]
Hello @ripperi8u 
Thanks for your report!

Created #5646 for this task.

Regards, Dmitriy
